### PR TITLE
Bump package versions for release

### PR DESCRIPTION
**Pull request by louwie17**
Bump versions in packages that had unreleased changes.
I also added `sideEffects` to the experimental library to allow for tree shaking (by default).

No changelog necessary.

## Test instructions

Read through the changes, make sure they make sense

### CODE REVIEW

File: packages/dependency-extraction-webpack-plugin/package.json
Comment (psealock) [inline packages/dependency-extraction-webpack-plugin/package.json:3]: DEWP will also need to be bumped here
https://github.com/woocommerce/woocommerce-admin/blob/14e3ace58b19c71b55469fb1ee0c5e7e65a3e6f4/bin/starter-pack/_package.json#L25

File: packages/components/CHANGELOG.md
Comment (psealock) [inline packages/components/CHANGELOG.md:1]: 🎉

**Review by psealock — APPROVED** [review]
Thanks @louwie17 !

I just noted one more place to update Dep Extr Webpack Plugin. And I should also note that the folks in Rubik will be interested to know when tree shaking is available 🎉 

One more thing, ExPlat versions were bumped already but never released so if you do a release that one will be included as well.

**Comment by louwie17** [comment]
Given the recent influx of PRs that pertain to the components package (thanks @samueljseay ), I will hold off on merging this until those are merged. As it would be great to have them be part of the `7.0` release.

**Comment by joelclimbsthings** [comment]
Quick note here, we're keen to publish the new `onboarding` package as well, but have a few more small [changes in flight](https://github.com/woocommerce/woocommerce-admin/issues/6966). May want to coordinate if that seems doable @louwie17 .

**Comment by louwie17** [comment]
> Quick note here, we're keen to publish the new `onboarding` package as well, but have a few more small [changes in flight](https://github.com/woocommerce/woocommerce-admin/issues/6966). May want to coordinate if that seems doable @louwie17 .

Yeah that's ok with me @joelclimbsthings, I currently was thinking early next week if that seems ok? There was quite a few recent PR's affecting packages, so I thought I hold of a couple day's.

**Comment by joelclimbsthings** [comment]
>Yeah that's ok with me @joelclimbsthings, I currently was thinking early next week if that seems ok? There was quite a few recent PR's affecting packages, so I thought I hold of a couple day's.

That's perfect, ours should be at the ready sometime tomorrow. Thanks @louwie17 !

**Review by psealock — APPROVED** [review]
Changes are looking good, and `onboarding` package is ready as well 🚀 Thanks @louwie17 !

**Pull request merged**
